airbyte-cdk/bulk/core/load/src/test/kotlin/io/airbyte/cdk/load/data/UnionTypeToDisjointRecordTest.kt

```diff
@@ -33,7 +33,7 @@ class UnionTypeToDisjointRecordTest {
     fun testUnionOfTypesWithSameNameThrows() {
         val (inputSchema, _) =
             SchemaRecordBuilder<Root>()
-                .with(UnionType.of(ObjectType(linkedMapOf()), ObjectTypeWithoutSchema))
+                .with(UnionType.of(StringType, ObjectTypeWithoutSchema))
```

Comment:
Curious why change this?

Reply:
This uses the names that correspond to the types they will use in avro/parquet

So: ObjectTypeWithoutSchema -> 'string'. I'll change this so it's more explicit.